app/serializers/resource_serializer.rb

```diff
@@ -1,10 +1,20 @@
 # frozen_string_literal: true
 
 class ResourceSerializer < ActiveModel::Serializer
+  type 'resource'
   attributes :id, :name, :abbreviation, :onesky_project_id
 
   belongs_to :system
 
   has_many :translations
   has_many :pages
+
+  has_many :latest_translations
+
+  def latest_translations
+    Translation.find_by_sql("select T.* from translations T inner join (select language_id, resource_id, max(version) as
```

Comment:
No, I'm not sure I need raw SQL.  The purpose is to return the translation with the latest version for each language.